CHANGELOG.rst

```diff
@@ -6,6 +6,12 @@ Changelog
 
 .. rst-class:: emphasize-children
 
+0.19.0
+------
+Added
+^^^^^
+- Added psycopg backend support
+
```

Comment:
Maybe we need add it in 0.18.2 section, this PR break  just add new feature

Reply:
I was assuming you were using Semantic Versioning (https://semver.org/) which indicates that a *.*.x is a bugfix and a *.x.* is backwards compatible functionality. But Either way is fine :)

Reply:
Not entirely, `*.*.x` mean user can upgrade to this version without change anything